src/windows/models/properties_model.py

```diff
@@ -267,9 +279,14 @@ def color_update(self, item, new_color, interpolation=-1, interpolation_details=
                 c = Effect.get(id=clip_id)
 
             if c:
+                # Create reference 
+                clip_data = c.data
+                if object_id:
+                    clip_data = c.data.get('objects').get(object_id)
+
                 # Update clip attribute
-                if property_key in c.data:
-                    log.debug("color update: %s" % c.data)
+                if property_key in clip_data:
+                    log.debug("color update: %s" % clip_data)
```

Comment:
(I know it was in the original code, but...)
```suggestion
                    log.debug("color update: %s", clip_data)
```
It's always better to pass log data as separate arguments, to take advantage of Python's lazy evaluation. If logging (for that level) is disabled, it doesn't even need to build the output string. When passing a single `%`-formatted string, it always has to build the string before calling the log function, even if it does nothing.

Reply:
Hmm. These ended up out of order due to later editing, so make that "same objection as below..."